src/reports/index.js

```diff
@@ -1,2 +1,2 @@
-export { default as InTransitItemReport } from './InTransitItemReport';
+export { default as InTransitItemsReport } from './InTransitItemReport';

```

Comment:
@Stanislau-Snapok could you please rename the file to `InTransitItemsReport` as well?